common/lib/xmodule/xmodule/css/video/display.scss

```diff
@@ -527,48 +542,91 @@ div.video {
           }
         }
 
-        a.add-fullscreen {
-          @extend %video-button;
-          background: url(../images/fullscreen.png) center no-repeat;
-          border-left: none;
+        .size-quality-controls {
+          display: block;
           float: left;
-          padding: 0 11px;
-          width: 30px;
-        }
+          margin: 0;
+          padding: 0;
+          list-style: none;
+          border-right: 1px solid #666;
 
+          .size-control {
+            margin: 0;
+            height: 46px;
+            border: none;
+
+            .add-fullscreen {
+              @extend %video-button;
+              display: inline-block;
+              width: 24px;
+              padding: 0 11px;
+              border: none;
+              line-height: 46px;
+              text-align: center;
+            }
+          }
 
-        a.quality-control {
-          @extend %video-button;
-          background: url(../images/hd.png) center no-repeat;
-          border-left: none;
-          float: left;
-          padding: 0 11px;
-          width: 30px;
+          .quality-control {
+            margin: 0;
+            height: 46px;
 
-          &.active {
-            background-color: #F44;
-            color: #0ff;
-            text-decoration: none;
-          }
+            .quality-toggle-control {
+              @extend %video-button;
+              float: left;
+              width: 30px;
+              padding: 0 11px;
+              border: none;
+
+              &.active {
+                background-color: #F44;
```

Comment:
Whoa what are these colors? Not a blocker but we should standardize/variable-ize these (if they are used... turquoise and red?!).